### DISCUSSION TITLE
Tip for using two root elements in <template>

### DISCUSSION THREAD

**Discussion by mimecine** [alpinejs/alpine#4422] [4. General]
You might already know, but I thought this was nifty: 
To get around the limit of one element in a template tag, I first tried the empty tag trick - <><el /><el /></> - which didn't work, but <x><el /><el /></x> did:)

**Comment by ekwoka** [comment]
`<>` is not an empty tag. It's just nothing.

You can get, for the vast majority of cases, the same result by just having an element with `display: contents` as the root, with the other elements inside. This makes it be the root for the purposes of the node tree, but for css layout, it's much like a fragment.
